rotkehlchen/db/filtering.py

```diff
@@ -721,8 +761,17 @@ def make(
         )
         filter_query = cast('HistoryEventFilterQuery', filter_query)
         filters: List[DBFilter] = []
-        if asset is not None:
-            filters.append(DBAssetFilter(and_op=True, asset=asset, asset_key='asset'))
+        if assets is not None:
+            if len(assets) == 1:
+                filters.append(DBAssetFilter(and_op=True, asset=assets[0], asset_key='asset'))
+            else:
+                filters.append(
+                    DBMultiStringFilter(
+                        and_op=True,
```

Comment:
here three!

Hmmm I think what would make sense after all is to make sure that this is not given to the ctor at all. And if absolutely needed make a special ctor for this filter which omits it. We should not have arguments that make no sense.